### DISCUSSION TITLE
Updates??

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#4665] [General]
Haven't seen the shadow of an update since Aug. 07.  Have I missed something?  Have you changed the procedure somehow and I missed the memo?

Cheers;
Speedy

**Comment by BusterNeece** [comment]
Version 0.14.1 was released on August 22. You can see release information on our Changelog:
https://github.com/AzuraCast/AzuraCast/blob/main/CHANGELOG.md

If you're not getting update notifications in your installation, there are many possible causes for this and it should be created as a GitHub issue, complete with the requisite logs.

**Reply by SpeedyCharly** [reply]
Ok Thks SlvrEagle - did a rolling update - will get back with an issue report if updates still don't show in the next few weeks...
